app/server-utils/stampy.ts

```diff
@@ -452,3 +452,5 @@ export const incAnswerColumn = async (column: string, pageid: PageId, subtract:
 
 export const makeColumnIncrementer = (column: string) => (pageid: PageId, subtract: boolean) =>
   incAnswerColumn(column, pageid, subtract)
+export const makeColumnDecrementer = (column: string) => (pageid: PageId, subtract: boolean) =>
+  incAnswerColumn(column, pageid, !subtract)
```

Comment:
eeh, `!subtract`?!? 😖

TBH, I would prefer to have 2 columns in Coda to track helpful and unhelpful (`2 - 1` has a different vibe from `100 - 99`), but if it's already decided that we want only 1 column please get rid of the `subtract` parameter from all places, replacing it with a `decAnswerColumn` function (or some better way)

Reply:
There are now 2 columns